docs/UserGuide.md

```diff
@@ -175,6 +175,12 @@ Examples:
 * `addtask d/Remind Alice to change bedsheets. r/5` Adds a task with description "Remind Alice to change bedsheets." to Room #5.
 * `addtask d/Running low on masks and needs to be restocked. r/1 dd/12-1-2021` Adds a task with description "Running low on masks and needs to be restocked." and due date "12 Jan 2021 at 12am" to Room #1.
 
+### 3.9 Lists the current rooms: `listRoom`
+
+Lists all the current available rooms.
```

Comment:
Do you think it is better to write "Lists all the rooms in the application." instead? I got confused with the word 'available' thinking it meant like empty rooms without patients.

Reply:
Okay I will change it.